src/main/java/org/tron/core/db/api/index/TransactionIndex.java

```diff
@@ -27,6 +27,7 @@
   public static Attribute<WrappedByteArray, String> OWNERS;
   public static Attribute<WrappedByteArray, String> TOS;
   public static Attribute<WrappedByteArray, Long> TIMESTAMP;
+  public static Attribute<WrappedByteArray, Long> INDEX_CREATE_TIMESTAMP;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this "public static INDEX_CREATE_TIMESTAMP" field final ([squid:S1444](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1444))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1276&issues=AWUdXkliyxJLWg291tgX&open=AWUdXkliyxJLWg291tgX)

[ISSUE_KEY]: AWUdXkliyxJLWg291tgX